giraffe/package.json

```diff
@@ -107,6 +107,7 @@
     "react-dom": "^16.8.0"
   },
   "dependencies": {
-    "merge-images": "^2.0.0"
+    "merge-images": "^2.0.0",
+    "pretty-quick": "^3.1.0"
```

Comment:
This should be in `devDependencies` just like `prettier` is.

Reply:
got it.  manually moved it over.